plugins/mcreator-localization/lang/texts.properties

```diff
@@ -937,6 +937,7 @@ blockly.block.entity_lookpos_y=Look Y position of %2 with raytrace distance %1 f
 blockly.block.entity_lookpos_z=Look Z position of %2 with raytrace distance %1 fluid mode %3 block mode %4
 blockly.block.entity_makeride=Make %1 ride %2
 blockly.block.entity_maketamed=Tame %1 by %2
+blockly.block.entity_makestopriding=Make %1 stop riding the current entity
```

Comment:
```suggestion
blockly.block.entity_makestopriding=Make %1 stop riding its current vehicle
```

Reply:
We should keep `entity` instead of `vehicle` as we don't use it for other blocks.